libraries/classes/Server/Privileges.php

```diff
@@ -1376,7 +1378,7 @@ private function getGlobalAndDatabasePrivileges(string $db): array
             (
                 SELECT `User`, `Host`, ' . $listOfPrivileges . ' `Db`, \'d\' AS `Type`
                 FROM `mysql`.`db`
-                WHERE \'' . $this->dbi->escapeString($db) . '\' LIKE `Db` AND NOT (' . $listOfComparedPrivileges . ')
+                WHERE \'' . $this->dbi->escapeString($db->getName()) . '\' LIKE `Db` AND NOT (' . $listOfComparedPrivileges . ')
```

Comment:
The length of the line exceeds 120 characters.
```suggestion
                WHERE \'' . $this->dbi->escapeString($db->getName()) . '\' LIKE `Db` AND NOT ('
                . $listOfComparedPrivileges . ')
```